adaptors/plugins.js

```diff
@@ -1,4 +1,6 @@
+"use strict";
+
 module.exports = [
-  require('lout'),
-  require('hapi-auth-bearer-token')
+  require(`lout`),
```

Comment:
yes, I believe the exported array is directly passed to hapi's register function